### PR TITLE
Bugfix fix Card Container alignment on Get Started page

### PR DESCRIPTION
**Pull request by mukhtyar**

### CODE REVIEW

File: src/components/cards/CardsContainer.svelte [hunk 1]
```diff
@@ -23,6 +23,9 @@
   }
 </style>
 
-<ul style="--gap:{gridGap}rem; --card-width:{cardWidth}rem">
+<ul
+  class="card--list--container"
```
Comment (mukhtyar) [inline src/components/cards/CardsContainer.svelte]: added a class here to target the `ul` element containg cards in `_carbon-overrides.scss`

File: src/scss/site/_carbon-overrides.scss [hunk 2]
```diff
@@ -46,7 +46,7 @@
 }
 
 .content {
-  ul:not(.bx--tabs__nav),
+  ul:not(.bx--tabs__nav):not(.card--list--container),
```
Comment (mukhtyar) [inline src/scss/site/_carbon-overrides.scss]: This removes `padding-left` for the `ul` element containing the cards. Removing  `padding-left` makes the cards align with the header on all viewport sizes

File: src/components/cards/CardsContainer.svelte (hunk 1 above)
Comment (clhenrick) [reply src/components/cards/CardsContainer.svelte]: @mukhtyar could we make the class name more similar to the Component name? Maybe `cards-container-list`? That way it's a little easier to find what it relates to in the Carbon style overrides `.scss` partial.

**Review by clhenrick — COMMENTED** [review]
@mukhtyar thanks for debugging this. I just added a suggestion to change the class name you added to the `CardsContainer`'s `<ul>`

File: src/components/cards/CardsContainer.svelte (hunk 1 above)
Comment (mukhtyar) [reply src/components/cards/CardsContainer.svelte]: updated class name in https://github.com/berkeley-gif/caladapt-website-2021/pull/148/commits/a9a6094666d89e45b8e526a6843f71572bb55959

**Pull request merged**